### ISSUE TITLE
First load of plugin configuration - error in JS

### ISSUE THREAD

**Issue by crobibero** [jellyfin/jellyfin-web#517]
**Describe the bug**
JS from Plugin configuration is run before loading required libraries

**To Reproduce**

1. Go to any plugin with configuration
2. `Dashboard.showLoadingMsg is not a function` error
3. Navigate away, then return to plugin configuration to having working JS

**Expected behavior**

Required JS libraries to be loaded before plugin JS is executed

**Logs**

```
triggering app resume event apphost.js:195:21
appRouter - processing path request /web/configurationpage appRouter.js:390:17
appRouter - user is authenticated appRouter.js:411:21
TypeError: Dashboard.showLoadingMsg is not a function
index.html:61:27
Password fields present on an insecure (http://) page. This is a security risk that allows user login credentials to be stolen.
index.html
appRouter - processing path request /web/installedplugins.html appRouter.js:390:17
appRouter - user is authenticated appRouter.js:411:21
Requesting url without automatic networking: http://192.168.59.109:8096/Plugins apiclientcore.js:264:21
Requesting url without automatic networking: http://192.168.59.109:8096/web/configurationpages?pageType=PluginConfiguration apiclientcore.js:264:21
appRouter - processing path request /web/configurationpage appRouter.js:390:17
appRouter - user is authenticated appRouter.js:411:21
Requesting url without automatic networking: http://192.168.59.109:8096/users apiclientcore.js:264:21
Password fields present on an insecure (http://) page. This is a security risk that allows user login credentials to be stolen.
index.html
Requesting url without automatic networking: http://192.168.59.109:8096/Plugins/94FB77C3-55AD-4C50-BF4E-4E5497467B79/Configuration apiclientcore.js:264:21
appRouter - processing path request /web/installedplugins.html appRouter.js:390:17
appRouter - user is authenticated appRouter.js:411:21
Requesting url without automatic networking: http://192.168.59.109:8096/Plugins apiclientcore.js:264:21
Requesting url without automatic networking: http://192.168.59.109:8096/web/configurationpages?pageType=PluginConfiguration apiclientcore.js:264:21
appRouter - processing path request /web/configurationpage appRouter.js:390:17
appRouter - user is authenticated appRouter.js:411:21
Requesting url without automatic networking: http://192.168.59.109:8096/users apiclientcore.js:264:21
Password fields present on an insecure (http://) page. This is a security risk that allows user login credentials to be stolen.
index.html
Requesting url without automatic networking: http://192.168.59.109:8096/Plugins/E29621A5-FA9E-4330-982E-EF6E54C0CAD2/Configuration apiclientcore.js:264:21
app is hidden apphost.js:203:21
```

**Screenshots**

Load 1:
![firefox_2019-10-10_08-03-31](https://user-images.githubusercontent.com/24963659/66576499-0f0fd280-eb35-11e9-9d0d-0fa854d968c8.png)
Load 2
![firefox_2019-10-10_08-04-13](https://user-images.githubusercontent.com/24963659/66576503-10d99600-eb35-11e9-88cd-635f8171bc71.png)

**System (please complete the following information):**
 - OS: Docker (nightly)
 - Browser: Firefox
 - Jellyfin Version: 10.4.0

**Comment by JustAMan** [comment]
paging @vitorsemeano @dkanada @thornbill

**Comment by dkanada** [comment]
That's related to [this](https://github.com/dkanada/jellyfin-web/blob/master/src/legacy/dashboard.js) legacy script but I don't think it was changed before the 10.4.0 release so it's not immediately clear what the issue could be.

**Comment by crobibero** [comment]
Fixed in 10.5
